### ISSUE TITLE
Ability to specify different input and output devices

### ISSUE THREAD

**Issue by mediadog** [tebjan/VVVV.Audio#132]
I have lots of cases where I want to take audio in from one device and send it out another, particularly when using Dante for network audio.  Could Engine be enhanced to take a device for input and output?

The most flexible,although requiring more patch wiring, would be for Engine nodes to output a context for other nodes to use so you could have (n) number of different input and output devices.  But just being able to specify one input and a different output would be a huge win.

**Comment by sunep** [comment]
Yes, that could be very handy, I have earlier worked with multiple interfaces to have the right amount of IO for the project
